.gitignore

```diff
@@ -1,6 +1,7 @@
 *.pyc
 *.pdf
 *.swp
+.idea/*
```

Comment:
revert? I think you are the only user who uses pycharm. Open to being convinced that it is really needed.